core/new-gui/src/app/workspace/service/workflow-graph/model/joint-graph-wrapper.ts

```diff
@@ -202,6 +204,12 @@ export class JointGraphWrapper {
     return jointLinkChangeStream;
   }
 
+  public getJointOperatorCellPostion(operatorID: string): Point | undefined {
+    const operatorCell = this.jointGraph.getCell(operatorID);
+    if (operatorCell.isElement()) {
+      return  {x: operatorCell.get('position').x, y: operatorCell.get('position').y};
+    }
+  }
```

Comment:
Use a new implementation defined in Simon's PR:

  `public getOperatorPosition(operatorID: string): Point {`
    `const cell: joint.dia.Cell | undefined = this.jointGraph.getCell(operatorID);`
    `if (! cell) {
      throw new Error(`opeartor with ID ${operatorID} doesn't exist`);
    }`
    `if (! cell.isElement()) {
      throw new Error(`${operatorID} is not an operator`);
    }`
    `const element  = <joint.dia.Element> cell;`
    `const position = element.position();`
    `return {
      x: position.x,
      y: position.y
    };`
  }

Reply:
Changed it